auditlog_ext_nrq/tests/test_auditlog_ext_nrq.py

```diff
@@ -64,3 +67,44 @@ def test_03_unlink_record(self):
             ('res_id', '=', self.translation_record.id),
         ], limit=1, order='id desc')
         self.assertEqual(log.log_category, 'unlink')
+
+    def test_04_create_vals(self):
```

Comment:
@YogeshMahera-SerpentCS It's hard for me to understand the scope of this test, and why the test is done in the way it's written here. Can't we simply feed create() method with specific `old_value_text` and `new_value_text`, and check the converted values against expected results (specific times in Japan tz)?

Reply:
Hi @yostashiro yes right but, Here we do add test-case based on the original flow of audit log module. So if we miss something then it will clear. Also, you correct directly we can pass the value to create method.